tests/dragonfly/acl_family_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from redis import asyncio as aioredis
+from . import DflyInstanceFactory
+from .utility import disconnect_clients
+
+
+@pytest.mark.asyncio
+async def test_acl_list_default_user(df_factory: DflyInstanceFactory):
+    """
+    make sure that the default created user is printed correctly
+    """
+    instance = df_factory.create(port=1111)
+    instance.start()
+    client = aioredis.Redis(port=instance.port)
```

Comment:
you don't need a separate dfly instance for that, starting/stopping those takes time

you can use the `async_client` fixture as many others tests do (see connection_test.py for example), there is also the the `dfly_args()` decorator if you need to pass custom args

Reply:
brain farted here, nice catch